modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.tsx

```diff
@@ -194,14 +234,14 @@ const Main = ({
 		]
 	);
 
-	let newValue = value;
+	let newValue: string | string[] | undefined = valueArray; // aqui
```

Comment:
i think you forgot this comment here, could you please remove it?

Reply:
😮 Sorry for that! Thanks, @rebsilva!

Reply:
no problem!